vignettes/tcplfit2-vignette.Rmd

```diff
@@ -232,89 +269,95 @@ dat[wllt == "t" & acid %in% nconc[nconc > 1, acid],
 dat[wllt == "t" & acid %in% nconc[nconc == 1, acid],
     rpid := paste(acid, spid, wllt, srcf, "rep1", conc, sep = "_")]
 
-## Define rpid column for non-test compound wells
+# Define rpid column for non-test compound wells
 dat[wllt != "t",
     rpid := paste(acid, spid, wllt, srcf, apid, "rep1", conc, sep = "_")]
 
-## set repid based on rowid
+# set the replicate index (repi) based on rowid 
+# increment repi every time a replicate ID is duplicated
 dat[, dat_rpid := rowid(rpid)]
 dat[, rpid := sub("_rep[0-9]+.*", "",rpid, useBytes = TRUE)]
 dat[, rpid := paste0(rpid,"_rep",dat_rpid)]
 
-# Define concentration index
+# For each replicate, define concentration index
+# by ranking the unique concentrations
 indexfunc <- function(x) as.integer(rank(unique(x))[match(x, unique(x))])
 dat[ , cndx := indexfunc(conc), by = list(rpid)]
 ```
 
+```{r, echo=FALSE}
```

Comment:
This needs to be removed from the vignette.  An alternative solution for displaying the updated tcpl relevant tables is necessary.

Reply:
Another opportunity for API integration? @cthunes @brown-jason